src/main/java/io/temporal/samples/hello/HelloPolymorphicActivity.java

```diff
@@ -119,20 +119,18 @@
      *
      * <p>For this example we define two activity stubs, one for each of the defined activities.
      *
-     * <p>Let's take a look at each {@link ActivityOptions} defined: The "setScheduleToCloseTimeout"
-     * option sets the overall timeout that the workflow is willing to wait for activity to
-     * complete. For this example it is set to 2 seconds for each of the defined activities.
+     * <p>Let's take a look at each {@link ActivityOptions} defined: The "setStartToCloseTimeout"
```

Comment:
This same wording `let's take a look at each {@link ActivityOptions} defined:` in all of these is a slightly awkward.

Perhaps instead

`In the {@link ActivityOptions} definition the "setStartToCloseTimeout"... `